docs/public/_redirects

```diff
@@ -130,10 +131,10 @@ https://material-ui.dev/* https://material-ui.com/:splat 301!
 /api/expansion-panel-actions/ /api/accordion-actions/ 301
 /api/expansion-panel-details/ /api/accordion-details/ 301
 /api/expansion-panel-summary/ /api/accordion-summary/ 301
-/premium-themes* https://material-ui.com/store/ 301
+/premium-themes* https://mui.com/store/ 301
 /customization/themes/ /customization/theming/ 301
-https://v1-5-0.material-ui.com/* https://v1.material-ui.com/:splat 301!
-https://v3-9-0.material-ui.com/* https://v3.material-ui.com/:splat 301!
+https://v1-5-0.mui.com/* https://v1.mui.com/:splat 301!
```

Comment:
Should we remove these?

Reply:
No objections, the context: https://github.com/mui-org/material-ui/issues/16541, likely in a different PR. And likely we need to do the same for some of the v4 versions.

Reply:
Alright, will do it in a dedicated PR after the stable release.